code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -1099,6 +1099,18 @@
 	UnregisterSignal(owner, COMSIG_ATOM_UPDATE_OVERLAYS)
 	owner.update_icon()
 
+/datum/status_effect/ineffective_thermoregulation
+	id = "no_temp_reg"
+	alert_type = /atom/movable/screen/alert/status_effect/no_temp_reg
+	duration = 2 SECONDS
```

Comment:
2 seconds on a replace status effect seems pretty negligible as an effect. You've knocked off some of the power on the projectiles, so there is presumably some expectation that you're going to be constantly shooting the target, but temp guns are more closer to a 'setup' tool. You're going to stop firing it at some point. And once you do, the temps going to normalize fairly fast.

I suggest having a bit of a higher duration if it's going to be using STATUS_EFFECT_REPLACE